app/assets/javascripts/miq_application.js

```diff
@@ -1339,6 +1339,43 @@ function miqInitSelectPicker() {
   $('.bootstrap-select > button[title]').not('.selectpicker').tooltip({container: 'none'});
 }
 
+function miqInitCodemirror(options) {
+  if (! miqDomElementExists(options.text_area_id)) {
+    return;
+  }
+
+  var textarea = $('#' + options.text_area_id)[0];
+
+  ManageIQ.editor = CodeMirror.fromTextArea(textarea, {
+    mode: options.mode,
+    lineNumbers: options.line_numbers,
+    matchBrackets: true,
+    theme: 'eclipse',
+    readOnly: options.read_only ? 'nocursor' : false,
+    viewportMargin: Infinity,
```

Comment:
Confirming, we don't support those IE versions anymore, so I'd actually encourage trailing commas :-)